bin/asc.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const [ nodePath, thisPath, ...args ] = process.argv;
+const nodeArgs = process.execArgv;
+
+const hasSourceMaps = nodeArgs.includes("--enable-source-maps");
+const posCustomArgs = args.indexOf("--");
+const isDeno = typeof Deno !== "undefined";
+
+if (isDeno) {
+  process.on = function() { /* suppress 'not implemented' message */ };
+}
+
+if ((!hasSourceMaps || ~posCustomArgs) && !isDeno) {
```

Comment:
What does `pos` stand for?

Reply:
Oh, piece of poo

Reply:
Stands for position here. Would you like it to be changed?